packages/central-server/package.json

```diff
@@ -46,6 +46,7 @@
     "case": "^1.6.1",
     "compare-versions": "^6.1.0",
     "cors": "^2.8.5",
+    "countries-and-timezones": "^3.6.0",
```

Comment:
I toyed with making our own dictionary of timezones to country names, but this library is pretty minimal and using it means we aren't responsible for any ISO standard updates etc

Reply:
yes I agree that it's good to used an established package even though it's pretty simple